header.php

```diff
@@ -23,7 +23,7 @@
 // to a valid X-Frame-Options header value or false
 if (defined('FORUM_FRAME_OPTIONS'))
 {
-	if (preg_match('/^(?:allow-from|deny|sameorigin)/i', FORUM_FRAME_OPTIONS) === 1)
+	if (preg_match('%^(?:allow-from|deny|sameorigin)%i', FORUM_FRAME_OPTIONS))
```

Comment:
We seem to be using `%` and `/` as delimiters [very inconsistently](https://github.com/fluxbb/fluxbb/search?utf8=%E2%9C%93&q=preg_match) throughout the codebase.
Perhaps a ticket should be created to pick either one and implement it everywhere.